integration-tests/universal/log_poller/helpers.go

```diff
@@ -1068,9 +1068,20 @@ func setupLogPollerTestDocker(
 		return network
 	}
 
+	ethBuilder := ctf_test_env.NewEthereumNetworkBuilder(t)
+	err = ethBuilder.
+		WithConsensusType(ctf_test_env.ConsensusType_PoS).
+		WithConsensusLayer(ctf_test_env.ConsensusLayer_Prysm).
+		WithExecutionLayer(ctf_test_env.ExecutionLayer_Geth).
+		WithBeaconChainConfig(ctf_test_env.BeaconChainConfig{
+			SecondsPerSlot: 4,
+			SlotsPerEpoch:  2,
+		}).
+		Build()
+
 	env, err = test_env.NewCLTestEnvBuilder().
 		WithTestLogger(t).
-		WithGeth().
+		WithPrivateEthereumNetwork(ethBuilder).
```

Comment:
@Tofel could you please check my comment [here](https://github.com/smartcontractkit/chainlink-testing-framework/pull/764/files#r1389082129)? Why not pass an `EthereumNetwork` object (can be an interface) here to `WithEthereumNetwork(ethNetwork)`?

Reply:
done